payment/src/main/java/com/kyriba/conference/payment/domain/PaymentMethodType.java

```diff
@@ -0,0 +1,9 @@
+package com.kyriba.conference.payment.domain;
+
+/**
+ * @author Igor Lizura
+ */
+public enum PaymentMethodType {
+    CREDIT_CARD,
+    WIRE_TRANSFER;
```

Comment:
; is not needed :)

Reply:
Please explain.

Reply:
; after the last enum value is not needed

Reply:
:)